pkg/virt-config/config-map.go

```diff
@@ -44,22 +44,25 @@ import (
 
 const (
 	configMapName                     = "kubevirt-config"
-	FeatureGatesKey                   = "feature-gates"
-	EmulatedMachinesKey               = "emulated-machines"
-	MachineTypeKey                    = "machine-type"
-	useEmulationKey                   = "debug.useEmulation"
-	ImagePullPolicyKey                = "dev.imagePullPolicy"
-	MigrationsConfigKey               = "migrations"
-	CpuModelKey                       = "default-cpu-model"
-	CpuRequestKey                     = "cpu-request"
-	MemoryOvercommitKey               = "memory-overcommit"
-	LessPVCSpaceTolerationKey         = "pvc-tolerate-less-space-up-to-percent"
-	NodeSelectorsKey                  = "node-selectors"
-	NetworkInterfaceKey               = "default-network-interface"
+	FeatureGatesKey                   = "featureGates"
```

Comment:
Maybe I miss something, but this would probably break all existing installations which have setting on kubevirt.

Reply:
We need to support both for some time. If no kubevirt yaml is presented in the config map, the config should still be taken as-is from the config map with the old names.

Reply:
Thank you @rmohr :)
I hope now it's OK.